svelte-kit-scss/package.json

```diff
@@ -41,6 +41,7 @@
     "@storybook/svelte": "6.5.14",
     "@storybook/testing-library": "0.0.13",
     "@sveltejs/adapter-auto": "1.0.0-next.90",
+    "@sveltejs/adapter-netlify": "^1.0.0-next.74",
```

Comment:
Did we need to add the `adapter-netlify` ?

https://kit.svelte.dev/docs/adapters

> SvelteKit offers a number of officially supported adapters.
> 
> You can deploy to the following platforms with the default adapter, adapter-auto:
> 
> [Cloudflare Pages](https://developers.cloudflare.com/pages/) via [adapter-cloudflare](https://github.com/sveltejs/kit/tree/master/packages/adapter-cloudflare)
> [Netlify](https://netlify.com/) via [adapter-netlify](https://github.com/sveltejs/kit/tree/master/packages/adapter-netlify)
> [Vercel](https://vercel.com/) via [adapter-vercel](https://github.com/sveltejs/kit/tree/master/packages/adapter-vercel)

Just curious why that didn't work?

Reply:
Yes that is what on the docs but even with the `@sveltejs/adaptor-auto` you need to install the `@sveltejs/adapter-netlify` or you gonna get this error on Netlify
![image](https://user-images.githubusercontent.com/49324987/207703328-4a4c2fa4-0885-4d99-a3ec-30c6359bf5ea.png)